light-client/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 name = "tendermint-light-client"
-version = "0.14.0"
+version = "0.15.0"
 authors = ["Romain Ruetschi <romain@informal.systems>"]
```

Comment:
@romac who would you like to add here? @brapse for sure, and who else?

Reply:
> $ git shortlog -n -s -- light-client

shows

```
    22  Romain Ruetschi
     6  Alexander Simmerl
     4  Ismail Khoffi
     1  Ethan Buchman
     1  Shivani Joshi
     1  yihuang
```